src/main/scala/com/gu/fulfilmentLookup/Lambda.scala

```diff
@@ -61,7 +63,14 @@ trait FulfilmentLookupLambda extends Logging {
           populateAddressRecord(rows, subIndex)
         )
         logger.info(s"Performed successful lookup for ${lookupRequest.subscriptionName} in $fileName. subInFile: $subInFile | subIndex: $subIndex")
-        LookupResponse(200, responseBodyAsString(responseBody))
+        caseService.raiseCase(config, lookupRequest, responseBody) match {
```

Comment:
my only question is should we show the lookup response before we raise the case?
i.e. either have a second button to click, or just to check the fulfilment status as soon as they log in, and then let them press the button to report it.